exercises/F-strings-methods/exercise2.js

```diff
@@ -1,3 +1,11 @@
-const name = " Daniel  ";
+const name = " Azan  ";
+let nametrim = name.trim();
+let nameLength = name.length;
```

Comment:
You should get the length of nameTrim rather than name

Reply:
Thank you for the feedback, I will be sure to fix it.